src/Config.php

```diff
@@ -4,18 +4,19 @@
 
 use Zend\Config\Config as ZendConfig;
 use Zend\Config\Reader\Xml;
-use Kobens\Core\Exception\LogicException;
 
 /**
  * Class Config
  * @package Kobens\Core
+ *
+ * TODO: getLogDir and getRootDir are starting to feel like they don't belong here. Single Purpose Object.
  */
 final class Config
 {
     /**
-     * @var bool
+     * @var Config
      */
-    private $initialized;
+    private static $self;
```

Comment:
I fucked up. I labeled this `$self` here and `$instance` below.

@jsfgreen.... I ran this code even once....

Reply:
Force pushed fix.